<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewProgramDialog</class>
 <widget class="QDialog" name="NewProgramDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>86</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>327</width>
    <height>86</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>327</width>
    <height>86</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Create new program</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>174</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameEdit">
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="maxLength">
         <number>20</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewProgramDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
